app/sections/view-integrations/macro.test.js

```diff
@@ -0,0 +1,101 @@
+import request from 'supertest';
+import cheerio from 'cheerio';
+import { createTestHarness } from '../../testUtils/testHarness';
+
+const macroWrapper = `{% from './sections/view-integrations/macro.njk' import viewIntegrations %}
+                        {{ viewIntegrations(params) }}`;
+
+describe('view-integrations', () => {
+  it('should render the integrations section if section data provided', (done) => {
+    const context = {
+      params: {
+        section: {
+          sections: {
+            'supplier-asserted-integrations': {},
+          },
+        },
+      },
+    };
+
+    const dummyApp = createTestHarness(macroWrapper, context);
+    request(dummyApp)
+      .get('/')
+      .then((res) => {
+        const $ = cheerio.load(res.text);
+
+        const integrations = $('[data-test-id="view-integrations"]');
+
+        expect(integrations.length).toEqual(1);
+
+        done();
+      });
+  });
+
+  it('should not render the integrations section if no section data provided', (done) => {
```

Comment:
add another test to check the section is not shown if some other section is provided that is not 'supplier-asserted-integrations'

Reply:
is that not already covered in the 'supplier-asserted-integrations' section?

Reply:
actually i see what you mean